### PR TITLE
replace positron-beta with positron in hyperlinks

### PR DESCRIPTION
**Pull request by just-cameron**
Minor quality of life changes. Some of the links in Positron are redirecting to 

https://github.com/posit-dev/positron-beta/discussions

instead of the 

https://github.com/posit-dev/positron/discussions

discussion page. I corrected these, as well as license pages.

I added some minor formatting of the welcome page code as well.

**Note**: [This GitHub action](https://github.com/cpfiffer/positron/blob/f7c259d68342290cb6a9660ad9495e2d1a167665/.github/workflows/build-push-beta.yml#L47) refers to the beta repo name as well, but I haven't changed it.

### CODE REVIEW

**Comment by jennybc** [comment]
Congrats @cpfiffer for what I think may be the first external pull request 🎉 

Some of the changes you make (for example changing double quotes to single) violate hard style conventions for us and touch whitespace in files related to indentation testing. Would you be willing to narrow this down to the specific thing you set out to fix, which is the broken hyperlinks? That will make it easier to merge.

File: src/vs/workbench/contrib/welcomeGettingStarted/browser/positronWelcomePageHelp.tsx [hunk 1]
```diff
@@ -4,32 +4,47 @@
  *--------------------------------------------------------------------------------------------*/
 
 // CSS.
-import 'vs/css!./media/positronGettingStarted';
```
Comment (petetronic) [inline src/vs/workbench/contrib/welcomeGettingStarted/browser/positronWelcomePageHelp.tsx:7]: Could you please revert the double quote usage as we use single quotes for ts strings. The double quotes will raise warnings complaining they should only be used on externalized strings.

File: extensions/positron-r/src/test/snapshots/indentation-snapshots.R [hunk 2]
```diff
@@ -25,7 +25,7 @@ data |>
 
 # ->
 data |>
```
Comment (petetronic) [inline extensions/positron-r/src/test/snapshots/indentation-snapshots.R:27]: I don't think these whitespace changes are expected, as this is part of a snapshot test case for indentation rules? @lionel- would know best.

**Review by petetronic — CHANGES_REQUESTED** [review]
Thank you so much for your PR! It looks like there is a CLA prompt to address too (if you need help, let us know), and some small feedback from me, and one question for @lionel-

File: extensions/positron-r/src/test/snapshots/indentation-cases.R [hunk 3]
```diff
@@ -187,7 +187,7 @@ foo("<>") +
 # ---
 # Breaking parentheses in a pipeline
 # https://github.com/posit-dev/positron/issues/2650
-# https://github.com/posit-dev/positron-beta/discussions/46
+# https://github.com/posit-dev/positron/discussions/46
```
[the remark this answers is not in the document]
Comment (cpfiffer) [reply extensions/positron-r/src/test/snapshots/indentation-cases.R:190]: Oh, remove the reference entirely?

File: extensions/positron-r/src/test/snapshots/indentation-snapshots.R [hunk 4]
```diff
@@ -347,7 +347,7 @@ foo(
 # ---
 # Breaking parentheses in a pipeline
 # https://github.com/posit-dev/positron/issues/2650
-# https://github.com/posit-dev/positron-beta/discussions/46
```
[the remark this answers is not in the document]
Comment (cpfiffer) [reply extensions/positron-r/src/test/snapshots/indentation-snapshots.R:350]: Ah okay perfect.

**Comment by cpfiffer** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by cpfiffer** [comment]
Give me a sec to revert the whitespace changes

File: src/vs/workbench/contrib/welcomeGettingStarted/browser/positronWelcomePageHelp.tsx (hunk 1 above)
Comment (cpfiffer) [reply src/vs/workbench/contrib/welcomeGettingStarted/browser/positronWelcomePageHelp.tsx]: Yep, fixed!

File: extensions/positron-r/src/test/snapshots/indentation-snapshots.R (hunk 2 above)
Comment (cpfiffer) [reply extensions/positron-r/src/test/snapshots/indentation-snapshots.R]: Fixed!

**Comment by cpfiffer** [comment]
I rode the whitespace strugglebus (thanks editor with autoformatting) and it has been fixed. This should be okay now.

**Comment by cpfiffer** [comment]
I'd also squash commit because I have goofy commit titles here

**Comment by petetronic** [comment]
Thank you! This looks good. As the first fork attempting a PR build on this repo, I see that such a run cannot access secrets. We may no longer need this token given the posit-dev/ark repo now public. I'll look into this.

**Review by petetronic — APPROVED** [review]

**Comment by wesm** [comment]
I rebased this just to get a clean build

**Pull request merged**